swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category = "", condition = 0.0):
+        super().__init__(category, condition)
+        self.category = "Clothing"
```

Comment:
Great use of inheritance here Libby!

We can further refactor this code and reduce the possibility of Clothing items from being reassigned to anything other than 'clothing' like so:

```suggestion
    def __init__(self, condition = 0.0):
        super().__init__(category = "Clothing" , condition)
        
```